content/en/blog/2020/custom-istio-dns-name/index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Customizing Istio DNS stub names"
+subtitle: Changing or adding stub domains to make ServiceEntry hostnames routable
+description: Customizing global or region hostnames as declared in ServiceEntry(s) for routability.
+publishdate: 2020-05-05
+attribution: "Christian Posta (Solo.io)"
+keywords: [istio, dns, mTLS, coredns]
+---
+
+Istio can be run across multiple clusters and has the necessary configuration APIs to support [shared control plane](/docs/setup/install/multicluster/shared/) or a non-shared, [replicated control model](/docs/setup/install/multicluster/gateways). The motivation for deploying across multiple clusters has been covered excellently by fellow Istio contributor [Vadim Eisenberg](https://twitter.com/vadimeisenberg) in the [Multi-Mesh Deployments for Isolation and Boundary Protection](/blog/2019/isolated-clusters/) blog.
```

Comment:
Anything in `[]` I believe is not spell checked.

You will see an overall theme in this review - we wish to bound the `.spelling` file changes.

Reply:
Yep, will remove from `.spelling`